templates/azure-functions/package.json

```
@@ -0,0 +1,25 @@
+{
+  "scripts": {
+    "build": "remix build",
+    "dev": "swa start ./public --run \"remix dev\" --api-location .",
+    "postinstall": "func init --worker-runtime node --language typescript"
+  },
+  "dependencies": {
+    "@remix-run/azure-functions": "*",
+    "@remix-run/react": "*",
```

We should add `@remix-run/node`

```suggestion
    "@remix-run/azure-functions": "*",
    "@remix-run/node": "*",
    "@remix-run/react": "*",
```